lib/plotting_functions.py

```diff
@@ -668,23 +679,28 @@ def make_polar_plot(wks, case_nickname, base_nickname,
     d1_region_mean, d1_region_max, d1_region_min = domain_stats(d1, domain)
     d2_region_mean, d2_region_max, d2_region_min = domain_stats(d2, domain)
     dif_region_mean, dif_region_max, dif_region_min = domain_stats(dif, domain)
+    pct_region_mean, pct_region_max, pct_region_min = domain_stats(pct, domain)
 
     #downsize to the specified region; makes plotting/rendering/saving much faster
     d1 = d1.sel(lat=slice(domain[2],domain[3]))
     d2 = d2.sel(lat=slice(domain[2],domain[3]))
     dif = dif.sel(lat=slice(domain[2],domain[3]))
+    pct = pct.sel(lat=slice(domain[2],domain[3]))
 
     # add cyclic point to the data for better-looking plot
     d1_cyclic, lon_cyclic = add_cyclic_point(d1, coord=d1.lon)
     d2_cyclic, _ = add_cyclic_point(d2, coord=d2.lon)  # since we can take difference, assume same longitude coord.
     dif_cyclic, _ = add_cyclic_point(dif, coord=dif.lon)
+    pct_cyclic, _ = add_cyclic_point(pct, coord=pct.lon)
 
     # -- deal with optional plotting arguments that might provide variable-dependent choices
 
     # determine levels & color normalization:
     minval    = np.min([np.min(d1), np.min(d2)])
     maxval    = np.max([np.max(d1), np.max(d2)])
     absmaxdif = np.max(np.abs(dif))
+    absmaxpct = np.max(np.abs(pct))
+    #print('absmaxpct: ',absmaxpct)
```

Comment:
Could you remove this commented out line?

Reply:
Done.